server/server.py

```diff
@@ -318,6 +328,29 @@ async def details(request: Request):
 
 
 def worker(sentence, speaker_id, model, use_aliases, new_speaker_ids):
+    def substitute_time(sentence):
+        # Regular expression to find time pattern (HH:MM)
+        time_pattern = re.compile(r'(\d{1,2}):(\d{2})')
+
+        # Find all matches of time pattern in the sentence
+        matches = re.findall(time_pattern, sentence)
+
+        if not matches:
+            return sentence
+
+        sentence = re.sub(r'les\s+', '', sentence, count=1)
+
+        # Iterate through matches and substitute with formatted time string
+        for match in matches:
+            H = int(match[0])
+            M = int(match[1])
+            dt = datetime.datetime(2017, 1, 31, H, M, 0, tzinfo=default_timezone())  # Using UTC timezone for simplicity
+            formatted_time = nice_time(dt, lang="ca")  # Assuming you have a function to format time in Catalan
```

Comment:
For the chosen behaviour this line should be `formatted_time = nice_time(dt, lang="ca", use_24hour=True)`

Reply:
Should be resolved in the latest commit.